README.md

```diff
@@ -83,6 +83,7 @@ default_migration_prefix | string | null | If specified, in the absence of the n
 generate_migration_name | bool | false | If enabled, a random migration name will be generated. The user will not be prompted for a migration name anymore. The parameter `default_migration_prefix` must be specified. The `--name` parameter can overwrite this setting.
 mark_generated_migration | bool | true | Enable or disable marking the migration as applied after creation.
 migration_base_class | string | '\Phinx\Migration\AbstractMigration' | Sets up base class of created migration.
+schema_file | string | `%%PHINX_CONFIG_DIR%%/db/migrations/schema.php` | Specifies the location for saving the schema file.
```

Comment:
The default is `schema.php`

Reply:
`schema_file` is the full path to the schema file

https://github.com/odan/phinx-migrations-generator/pull/74/files#diff-59a14b2d242c3a02b1f9a103d9e19956L121